app/src/main/java/org/p2p/wallet/home/ui/crypto/MyCryptoFragment.kt

```diff
@@ -109,13 +109,15 @@ class MyCryptoFragment :
     }
 
     private fun showTokenVisibilityStateChangeDialog(item: TokenCellModel) {
-        TokenVisibilityChangeBottomSheet.show(
-            fm = childFragmentManager,
-            token = item.payload,
-            isTokenHidden = item.isDefinitelyHidden,
-            requestKey = KEY_REQUEST,
-            resultKey = KEY_RESULT_TOKEN
-        )
+        if (item.isDefinitelyHidden || item.payload.canTokenBeHidden) {
```

Comment:
`item.isDefinitelyHidden` is not used for showing dialog. 
`canTokenBeHidden` should be used only

Reply:
`FIXED`